src/firefox/package-identifiers.js

```diff
@@ -0,0 +1,10 @@
+/* @flow */
+
+export const browserList = [
```

Comment:
Rename to `packageIdBases` ?

or just `export default`. The file name, `packageIdentifiers`, is already a nice name on its own.